docs/crossgen-scenarios.md

```diff
@@ -0,0 +1,50 @@
+
+# Crossgen Scenarios
+An introduction of how to run scenario tests can be found in [Scenarios Tests Guide](./scenarios-workflow.md).  The current document has specific instruction to run:
+
+- [Crossgen Throughput](#crossgen-throughput)
+- [Crossgen2 Throughput](#crossgen2-throughput)
+
+
+## Crossgen Throughput
+
+### Step 0 Generate Core Root
+Build CoreCLR in dotnet/runtime repo, which creates Core Root. Instruction can be found [here](https://github.com/dotnet/runtime/blob/master/docs/workflow/building/coreclr/README.md). A detailed introduction of Core Root can be found [here](https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/using-corerun.md).
```

Comment:
Is building tests also required? If so, it would be helpful to add clarification with [build tests](https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/coreclr/testing.md) link.